helpdesk/__manifest__.py

```diff
@@ -6,17 +6,20 @@
         Helpdesk""",
     'version': '11.0.1.0.0',
     'license': 'AGPL-3',
-    'author': 'Odoo Community Association (OCA), Domatix',
+    'author':   'AdaptiveCity, '
+                'C2i Change 2 Improve, '
+                'Domatix, '
+                'Factor Libre, '
+                'SDi Soluciones, '
+                'Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/helpdesk',
     'depends': [
         'mail',
-        'portal',
     ],
     'data': [
         'data/helpdesk_data.xml',
         'views/helpdesk_team_views.xml',
-        'security/helpdesk_security.xml',
-        'security/ir.model.access.csv',
```

Comment:
Don't delete these data files or Travis will fail. Also security will not be added.